service/src/main/java/skills/services/admin/InviteOnlyProjectService.groovy

```diff
@@ -206,6 +217,11 @@ class InviteOnlyProjectService {
             throw new SkillException("Invitation Code has already been used", projectId, null, ErrorCode.ClaimedInvitationCode)
         }
 
+        if (validateInviteEmail && userInfo?.email != projectAccessToken.recipientEmail) {
```

Comment:
since invite emails are user entered then email comparison must be case insensitive